modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/taglib/clay/data/set/provider/PreviewCommerceOrderItemDataSetDataProvider.java

```diff
@@ -160,6 +193,23 @@ private String _formatFinalPrice(
 		return StringPool.BLANK;
 	}
 
+	private String _formatImportDate(
+		Date requestedDeliveryDate, String requestedDeliveryDateFormat) {
+
+		try {
+			Format simpleDateFormat =
+				FastDateFormatFactoryUtil.getSimpleDateFormat(
+					requestedDeliveryDateFormat);
+
+			simpleDateFormat.parseObject(requestedDeliveryDate.toString());
+
+			return requestedDeliveryDate.toString();
+		}
+		catch (IllegalArgumentException | ParseException exception) {
+			return "";
```

Comment:
We should print and error message in this case, we could use this return statement to send an error message directly in the dataset table

Reply:
It must be a localized message